crates/wasmtime/src/linker.rs

```diff
@@ -485,4 +610,85 @@ impl Linker {
         }
         Ok(ret.clone())
     }
+
+    /// Returns the "default export" of a module.
+    ///
+    /// An export with an empty string is considered to be a "default export".
+    /// "_start" is also recognized for compatibility.
+    pub fn get_default(&self, module: &str) -> Result<Func> {
```

Comment:
This function feels a bit odd to me because the notion of a "default" export doesn't seems super well defined. Is there WASI documentation this could link to? 

In isolation it seems like this API should be on an `Instance` as well, but I understand why it's here instead of instances at this time. Perhaps they could share an implementation of sorts?

Reply:
Also since this is WASI-specific would it make sense to do something like `get_wasi_default()`?